packages/editor/src/client/components/CodeMirrorEditorMain.tsx

```diff
@@ -21,20 +21,18 @@ const additionalExtensions: Extension[] = [
 type Props = CodeMirrorEditorProps & {
   user?: IUserHasId,
   pageId?: string,
-  initialValue?: string,
-  isEditorMode: boolean,
+  isYjsEnabled: boolean,
   onEditorsUpdated?: (userList: IUserHasId[]) => void,
 }
 
 export const CodeMirrorEditorMain = (props: Props): JSX.Element => {
   const {
-    user, pageId, initialValue, isEditorMode,
-    onSave, onEditorsUpdated, ...otherProps
+    user, pageId, onSave, onEditorsUpdated, isYjsEnabled, ...otherProps
   } = props;
 
   const { data: codeMirrorEditor } = useCodeMirrorEditorIsolated(GlobalCodeMirrorEditorKey.MAIN);
 
-  useCollaborativeEditorMode(isEditorMode, user, pageId, initialValue, onEditorsUpdated, codeMirrorEditor);
+  useCollaborativeEditorMode(isYjsEnabled, user, pageId, onEditorsUpdated, codeMirrorEditor);
```

Comment:
責務の整理について悩みが生じるね。

useNonYjsModeEffect と useCollaborativeEditorMode は対になる存在なので、同じコンポーネントで宣言したい。
下の方にある以下の cmProps 作成部分も同様。
```
      // Disable the basic history configuration since this component uses Y.UndoManager instead
      basicSetup: {
        history: false,
      },
```

しかし useNonYjsModeEffect を packages/editor に持って行くことはできない。
以下のどちらかの手段を検討か

1. PageEditor に useNonYjsModeEffect と useCollaborativeEditorMode と `history: false` を持って行く
1. useNonYjsModeEffect と useCollaborativeEditorMode と `history: false` の処理を含み CodeMirrorEditorMain をレンダリングするコンポーネントを新規に apps/app に作成し、それを PageEditor から使う

Reply:
提案していただいた 案2 で実装しました
commit: https://github.com/weseek/growi/pull/8990/commits/5e89fff8ecc89ae52b17ad3e8a7e874bbc4f2609